lib/msf/core/module/stability.rb

```diff
@@ -0,0 +1,19 @@
+module Msf::Module::Stability
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def stability
+      instance = self.new
+      instance.notes['Stability'] ? instance.notes['Stability'] : []
```

Comment:
`instance.notes['Stability'] ? instance.notes['Stability'] : []` can be simplified to `instance.notes['Stability'] || []`, I think.